components/ArtPieceDetails/index.js

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+export default function ArtPieceDetails({ image, name, artist, year, genre }) {
+  const router = useRouter();
+  function handleBack() {
+    router.push("/art-pieces");
+  }
+  return (
+    <>
+      <section>
```

Comment:
a section normally has a heading, so you could think about making line 15 to be a heading instead of a small-tag